### DISCUSSION TITLE
控制台错乱

### DISCUSSION THREAD

**Discussion by suyllovecandy** [higress-group/higress#544] [Q&A] [answered]
![image](https://github.com/alibaba/higress/assets/49014244/bac46c26-8d9b-4eeb-9ed3-b459ed69ca76)

通过[环境二：脱离 K8s 在 Docker Compose 中使用](https://higress.io/zh-cn/docs/user/quickstart#%E7%8E%AF%E5%A2%83%E4%BA%8C%E8%84%B1%E7%A6%BB-k8s-%E5%9C%A8-docker-compose-%E4%B8%AD%E4%BD%BF%E7%94%A8) 方式安装的

**Comment by johnlanni** [comment]
@suyllovecandy 你的浏览器是？用chrome试试呢

**Comment by CH3CHO** [comment]
这是不是群里我回答过的那个问题诶？当时换了个浏览器就好了？

**Comment by suyllovecandy** [comment, marked as answer]
你好，是的浏览器版本太低，升级后没事了，这个关了吧，谢谢



---原始邮件---
发件人: "Kent ***@***.***&gt;
发送时间: 2023年9月25日(周一) 晚上9:56
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [alibaba/higress] 控制台错乱 (Discussion #544)




 
这是不是群里我回答过的那个问题诶？当时换了个浏览器就好了？
